packages/custom-functions-metadata-plugin/src/loader.js

```diff
@@ -0,0 +1,9 @@
+const { getOptions } = require('loader-utils');
+
+module.exports = function addFunctionAssociations(source) {
```

Comment:
Why is this function needed? It was created but it isn't called anywhere.

Reply:
It defines the loader, which is passed directly as a file reference here: https://github.com/OfficeDev/Office-Addin-Scripts/pull/422/files#diff-addd1af0a602b17de45903dbd9a9d3726d67753c6ac81a238e31dc21361860bdR45

This could be an anonymous export too, though. I just named it out of habit.